src/plugins/intel_gpu/src/kernel_selector/core/kernel_selector_params.cpp

```diff
@@ -414,15 +414,13 @@ bool ParamsKey::Support(const ParamsKey& k) const {
         return false;
     if (!((key.outputWeightsType.raw & k.key.outputWeightsType.raw) == k.key.outputWeightsType.raw))
         return false;
-    if (!((key.inputLayout & k.key.inputLayout) != 0 || key.inputLayout == k.key.inputLayout))
+    if (!(k.key.inputLayout.none() || (key.inputLayout & k.key.inputLayout).any()))
```

Comment:
why did you fixed this? please do not change common code if it is not critical for adding a new primitive...